extension.js

```diff
@@ -80,7 +80,12 @@ var AdBlocker = class AdBlocker {
         if (title === 'spotify' || title === 'advertisement') {
             this.mute();
         } else {
-            this.unmute();
+        
+        // Delay for 1 second to fix last second ad not muting
+        setTimeout(() => { 
```

Comment:
There's no `setTimeout` function on GNOME gjs, we should use the `GLib.timeout_add`, you can find some code that uses that in `gnome-shell`: https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/master/js/ui/extensionSystem.js#L50-53

Reply:
Ok Let me check